slave/server.py

```diff
@@ -106,10 +106,10 @@ def retrieveOffline(self):
 
     def captureResults(self, query):
         queryParsed = urlparse.parse_qs(query)
-        fp = open("slave/results", "w");
-        fp.write(queryParsed["results"][0]);
+        fp = open("slave/results", "w")
+        fp.write(queryParsed["results"][0])
         fp.close()
-        return False
+        return True
```

Comment:
Oh this is a very badly programmed. Sorry.
The "return False" signals here that the caller needs to report a 404:
( self.send_error(404, "File not found") )
since that url actually doesn't show anything currently. (which return True would signify.)

Maybe the actual fix here would be to do:

content = "results successfully captured"
self.send_response(200)
self.send_header("Content-Length", len(content))
self.wfile.write(bytes(content))
f.close()
return True;